desktop/core/src/desktop/celery.py

```diff
@@ -46,13 +47,19 @@ def setup_beat_schedule(self):
     self.conf.beat_schedule = {
       'check_disk_usage_and_clean_task': {
       'task': 'filebrowser.check_disk_usage_and_clean_task',
-      'schedule': 1000.0,  # Run every 1000 seconds
+      'schedule': TASK_SERVER_V2.CHECK_DISK_USAGE_AND_CLEAN_TASK_PERIODIC_INTERVAL.get(),
```

Comment:
This code should be part of hue.ini. It should not enabled automatically.

Reply:
Added a config `check_disk_usage_and_clean_task_enabled`, 'cleanup_stale_uploads_in_redis_enabled', which is set to False by default. So it wont be enabled automatically.